### PR TITLE
Link to Rabbit connection settings article

### PR DESCRIPTION
**Pull request by DavidBoike**
To avoid confusion, as the connection settings page is a little buried currently, and the existing text makes it sound like UseTransport is all that's required.

### CODE REVIEW

File: nservicebus/rabbitmq/index.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ To use RabbitMQ as the underlying transport:
 
 snippet:rabbitmq-config-basic
 
+The RabbitMQ transport requires a connection string to connect to the RabbitMQ broker. See [Connection settings](/nservicebus/rabbitmq/) for options on how to provide the connection string.
```
Comment (bording) [inline nservicebus/rabbitmq/index.md]: Feels like there is one too many "RabbitMQ"s in the sentence.

Comment (bording) [inline nservicebus/rabbitmq/index.md]: The link url should be `connection-settings.md`.

**Comment by DavidBoike** [comment]
@bording Oops, nice catch on the self-link. Better now?

**Comment by bording** [comment]
@DavidBoike I think the link could be be just "connection-settings.md" and not need the rest of the path, but I'll leave that to the @Particular/docs-maintainers to decide how they prefer it.

**Comment by SimonCropp** [comment]
Indifferent to link depth. Both will work and we verify them. The relative is a little more concise, but up to you

**Comment by DavidBoike** [comment]
I'll remove WIP and merge tonight during my talk.

**Comment by DavidBoike** [comment]
Didn't end up making it that far in my talk. :-( Oh well.

**Pull request merged**